src/backend/marsha/core/migrations/0028_auto_20211202_1319.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.2.9 on 2021-12-02 13:19
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def migrate_is_registered(apps, schema_editor):
+    """Migrate the live type info into the live_type field."""
+    Liveregistrations = apps.get_model("core", "Liveregistration")
+    liveregistration_to_migrate = Liveregistrations.objects.filter(is_registered=False)
+    for liveregistration in liveregistration_to_migrate:
+        liveregistration.is_registered = True
+        liveregistration.save()
```

Comment:
```suggestion
    Liveregistrations.objects.filter(is_registered=False).update(is_registered=True)
```
I'm not sure it will work on a migration context, but it may worth a try ?